prime-router/build.gradle.kts

```diff
@@ -881,7 +881,13 @@ dependencies {
     implementation("io.github.java-diff-utils:java-diff-utils:4.11")
     implementation("io.ktor:ktor-client-core:$ktorVersion")
     implementation("io.ktor:ktor-client-cio:$ktorVersion")
+    // the apache4 client will NOT free resources automatically between requests
```

Comment:
Before merging this PR, these comments should be removed and a ticket created instead to update other usages (wait until PR gets fully approved though in case something comes up)

Reply:
@david-navapbc Bumping this comment

Reply:
ack